controller/destination/server_test.go

```diff
@@ -1,124 +1,232 @@
 package destination
 
 import (
-	"fmt"
+	"context"
+	"reflect"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/pkg/errors"
```

Comment:
It looks like you're importing a 3rd-party lib when you could be using the "errors" package from stdlib. In fact, if you fix this import, then you don't need any of the `Gopkg.*` and deps changes on this branch.

Reply:
That was an intellij import mistake 🤡